### DISCUSSION TITLE
Any recommendations on which proxies to use for bypassing cloudflare?

### DISCUSSION THREAD

**Discussion by simplexx** [kaliiiiiiiiii/Selenium-Profiles#113] [General]
Hi! 
Due to work, I was not able to get anything done in the last year using Selenium Profiles, but soon I will have available time again and I want to deep dive into selenium profiles and selenium wire again :). Currently I am looking for a good reliable proxy service to use for bypassing cloudflare using selenium profiles. Do you have any recommendation related to this?
Cheers, Chris

**Comment by ttlns** [comment]
maybe you'll want to move to driverless

**Comment by ttlns** [comment]
@simplexx This is a warning!. Please don't open any issues unless there's q bug.
